packages/react/src/themes/teams/components/Flex/flexVariables.ts

```diff
@@ -0,0 +1,17 @@
+import { pxToRem } from '../../../../lib'
+import { FlexGap, FlexPadding } from '../../../../components/Flex/Flex'
+
+type GapValues = { [Key in FlexGap]: string }
+type PaddingValues = { [Key in FlexPadding]: string }
```

Comment:
We can use `Record` there, too 😺

Reply:
will change but, frankly, this type declaration seems to be more expressive to me - each time I see
```ts
Record<FlexGap, string >
```
it takes me some though effort to rearrange in my brain how resulting type will be composed.